### PR TITLE
Performance improvements to Cake line mappings

### PR DESCRIPTION
**Pull request by mholo65**
Some performance improvements noted by @DustinCampbell while he was looking through the Cake line mapping code.

### CODE REVIEW

File: OmniSharp.sln [hunk 1]
```diff
@@ -68,7 +68,7 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OmniSharp.Http.Driver", "sr
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OmniSharp.Stdio.Driver", "src\OmniSharp.Stdio.Driver\OmniSharp.Stdio.Driver.csproj", "{D2A78CEE-B278-476F-AF34-A7D6F792F973}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "OmniSharp.Script.Tests", "tests\OmniSharp.Script.Tests\OmniSharp.Script.Tests.csproj", "{9E4BA68C-7F4B-429A-A0C7-8CE7D41D610F}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OmniSharp.Script.Tests", "tests\OmniSharp.Script.Tests\OmniSharp.Script.Tests.csproj", "{9E4BA68C-7F4B-429A-A0C7-8CE7D41D610F}"
```
Comment (mholo65) [inline OmniSharp.sln]: No idea why this was changed?

**Comment by filipw** [comment]
@mholo65 same here 🙂

**Comment by mholo65** [comment]
@filipw yup, I guess I need to rebase and merge one at a time.. 😭

**Comment by mholo65** [comment]
@filipw fixed conflicts.

**Pull request merged**